src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -221,7 +224,7 @@ protected void shutdownService() {
       if(selector != null && selector.isOpen()) {
         selector.close();
       }
-    } catch (IOException e) {
+    } catch (Exception e) {
```

Comment:
It's your call...I probably would, _just_ in case things change in the future.

Another option would be to call into ExceptionUtils.handleException(e) so that if an ExceptionHandlerInterface is setup in the app it can at least log the exception.  Unless you truly want to hide that anything happened (I definetly agree we would not want to throw it here)